main.go

```diff
@@ -93,7 +93,8 @@ func main() {
 	// Setup exception tracking with Sentry if configured
 	if c.SentryDSN != "" {
 		if err = sentry.Init(sentry.ClientOptions{
-			Dsn: c.SentryDSN,
+			Dsn:          c.SentryDSN,
+			IgnoreErrors: []string{"401"},
```

Comment:
this only accept error of regex, but setting this up like this ignores 401 errors, tested out on free sentry account i made